src/main/java/tech/andreagreco/AnotherExample.java

```diff
@@ -0,0 +1,36 @@
+package tech.andreagreco;
+
+import java.math.BigDecimal;
+import java.util.Deque;
+import java.util.concurrent.Executors;
+import java.util.concurrent.LinkedBlockingDeque;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+public class AnotherExample {
+    
+    public void function() {
```

Comment:
For me it's a bit vague to add "function" as name of a method. Could you please add some more descriptive name? (It applies to the classnames too)